client/src/charts/wrapped_nivo/nivo_bar.js

```diff
@@ -127,7 +126,7 @@ export class NivoResponsiveBar extends React.Component {
             motionStiffness,
           }}
           legends={fix_legend_symbols(legends)}
-          keys={_.union([""], keys)}
+          keys={keys}
```

Comment:
Are negative bars covered now?

Reply:
Seems like it! Locally and on Dev link the negative bars are appearing